fem/nonlinearform.hpp

```diff
@@ -199,24 +199,41 @@ class BlockNonlinearForm : public Operator
        GridFunction-like block-vector data (e.g. in parallel). */
    mutable BlockVector xs, ys;
 
-   mutable Array2D<SparseMatrix*> Grads;
+   mutable Array2D<SparseMatrix*> Grads, cGrads;
```

Comment:
what does `cGrads` stand for? Conforming grads? I think a more verbose name would be useful.